backend/src/main/resources/db/migration/V1_0_11__Update_Information_About_Profile.sql

```diff
@@ -0,0 +1,29 @@
+ALTER TABLE credentials
+    ADD COLUMN phone_number NUMERIC(9,0) NULL;
+ALTER TABLE credentials
+    ADD COLUMN birth_date DATE NULL;
+
+ALTER TABLE locations
+    ADD COLUMN zip_code CHAR(5) NOT NULL;
+ALTER TABLE locations
+    ADD COLUMN island VARCHAR(15) NOT NULL;
+ALTER TABLE locations
+    ALTER COLUMN province DROP NOT NULL;
+ALTER TABLE locations
+    ALTER COLUMN town DROP NOT NULL;
+ALTER TABLE locations
+    ALTER COLUMN address DROP NOT NULL;
+
+ALTER TABLE volunteers
+    DROP COLUMN age;
+
+ALTER TABLE volunteers
+    ADD COLUMN twitter VARCHAR(255) NULL;
+ALTER TABLE volunteers
+    ADD COLUMN instagram VARCHAR(255) NULL;
+ALTER TABLE volunteers
+    ADD COLUMN linkedin VARCHAR(255) NULL;
+ALTER TABLE volunteers
+    ADD COLUMN photo VARCHAR(255);
+ALTER TABLE volunteers
+    ADD COLUMN additional_information VARCHAR(500);
```

Comment:
ValueObject?

Reply:
Hi Ibai, I don't think an object value is needed for those values. If you think otherwise please tell me.

Reply:
It's done. I use a valueObject to validate the field.